src/tailscale_svc_lb_controller/config.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+import logging
+import re
+import sys
+from os import environ as env
+
+# -- Constants
+CONTROLLER_PREFIX = "svc-lb.tailscale.iptables.sh"
+SERVICE_NAME_LABEL = CONTROLLER_PREFIX + "/svc-name"
+SERVICE_NAMESPACE_LABEL = CONTROLLER_PREFIX + "/svc-namespace"
+
+# -- Proxy Deployment Configuration
+RESOURCE_PREFIX = env.get("RESOURCE_PREFIX", "ts-")
+SECRET_NAME = env.get("SECRET_NAME", "tailscale-svc-lb")
+# LoadBalancerClass this instance of Tailscale Proxy will implement
+LOAD_BALANCER_CLASS = env.get("LOAD_BALANCER_CLASS", CONTROLLER_PREFIX + "/lb")
+# Label to use when selecting nodes for the Tailscale Proxy to run on. The value of this label should be 'true'
+NODE_SELECTOR_LABEL = env.get("TS_PROXY_NODE_SELECTOR_LABEL", None)
+
+# A semi-colon seperated string containing the names of any secrets that should be used
+# when pulling images. Secret must already exist and be present in the TS_PROXY_NAMESPACE
+IMAGE_PULL_SECRETS = env.get("IMAGE_PULL_SECRETS", "")
+if not re.match(r"^([a-z]|-|\d|;)*$", IMAGE_PULL_SECRETS):
```

Comment:
NIT: If you want, you can make the regexp a bit easier to read by dumping everything into the set:
```suggestion
if not re.match(r"^[a-z\d;-]*$", IMAGE_PULL_SECRETS):
```